etc/skydive.yml.default

```diff
@@ -100,16 +100,6 @@ agent:
       # delay in seconds between two metric updates
       # metrics_update: 30
   flow:
-    # Probes used to capture traffic.
```

Comment:
arf sorry, forgot that but there is many other places, check in contrib folder and the devstack plugin